python/src/scipp/_bins.py

```diff
@@ -158,8 +158,9 @@ def histogram(x, bins):
     return _call_cpp_func(_cpp.histogram, x, bins)
 
 
-def bin(x, edges=[], groups=[]):
+def bin(x, edges=[], groups=[], clear=[]):
```

Comment:
Maybe `erase`?

Reply:
Sure, that is consistent with the `BinBuilder` at least